config.toml

```diff
@@ -62,9 +62,9 @@ relativeURLs = true
              weight = 1
              parent="doc_features"
       [[menu.main]]
-          identifier = "doc_gettingstarted"
+          identifier = "doc_installation_guide"
           name = "Getting Started"
-          url = "/documentation/latest/getting-started"
+          url = "/documentation/latest/installation-guide"
```

Comment:
You probably should put a redirection on the old "getting-started" link.

I'm checking my old medium posts and in several posts I used the old (and now non-existent) https://kiali.io/documentation/getting-started/#_quick_install link. This is not your fault, but removing the `/documentation/latest/getting-started` page will *again* break external links.

In general, we should do a better job at not breaking external links. In particular, getting started or install pages are pretty important and we should try to never break potential external refferals.

Reply:
Good point. I'll see what I can do. I see I can add alias however they are directed to the edge version. Have to get more familiar with the whole versioning system. :+1:

Reply:
You can direct the link to latest, probably on config.toml.

Reply:
I've put some alias to restore previous broken links and the one that will be broken bc of this PR.